client/src/modules/list/components/ItemsList/actions.js

```diff
@@ -54,11 +54,11 @@ export const toggle = (
     });
 };
 
-export const vote = (itemId, listId, voterIds) => dispatch => {
+export const vote = (itemId, listId, didCurrentUserVoted) => dispatch => {
   dispatch(voteForItemRequest());
-  return patchData(`${ENDPOINT_URL}/lists/${listId}/update-item`, {
-    itemId,
-    voterIds
+  return patchData(`${ENDPOINT_URL}/lists/${listId}/vote-for-item`, {
+    didCurrentUserVoted,
```

Comment:
please do not use `true/false` flag - just create another api route e.g. `clear-vote` and `set-vote`